src/extension/features/accounts/change-enter-behavior/index.js

```diff
@@ -13,7 +13,7 @@ export class ChangeEnterBehavior extends Feature {
     const $outflowInput = $('.ynab-grid-cell-outflow input', $addRow);
     const $inflowInput = $('.ynab-grid-cell-inflow input', $addRow);
 
-    if (!$memoInput[0].getAttribute('data-toolkit-save-behavior')) {
+    if (!$memoInput[10].getAttribute('data-toolkit-save-behavior')) {
```

Comment:
Is this really meant to be `[10]` here? I'd expect `[0]`.

Reply:
Maybe something left over from forcing an error in a feature to see how the try/catch handled it? but yeah, seems odd otherwise.

Reply:
l m a o